crates/autopilot/src/run_loop.rs

```diff
@@ -176,6 +167,22 @@ impl RunLoop {
             );
         }
 
+        let considered_orders: HashSet<_> = solutions
+            .iter()
+            .flat_map(|solution| solution.solution.order_ids().copied())
+            .collect();
+        let winning_orders: HashSet<_> = solutions
+            .last()
+            .into_iter()
+            .flat_map(|solution| solution.solution.order_ids().copied())
+            .collect();
+        let considered_orders: Vec<_> = considered_orders
+            .difference(&winning_orders)
+            .cloned()
+            .collect();
+        self.persistence
+            .store_order_events(considered_orders, OrderEventLabel::Considered);
```

Comment:
Maybe we should store the `OrderEventLabel::Executing` here as well to make sure we have consistency (now it's still possible that reveal fails and an order that was part of every solution isn't labeled as having a solutions)